keyboards/ploopyco/trackball_thumb/info.json

```diff
@@ -0,0 +1,36 @@
+{
+    "keyboard_name": "PloopyCo Thumb Trackball",
+    "url": "www.ploopy.co",
+    "maintainer": "ploopyco",
+    "manufacturer": "Ploopy Corporation",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "usb": {
+        "vid": "0x5043",
+        "pid": "0x5C46",
+        "max_power": 100
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "pointing_device": true,
+        "encoder": false
```

Comment:
Why doesn't the optical encoder on the scroll wheel count as an `encoder` here?

Reply:
basically, until the encoder feature supports other ways to configure the inputs, it's using a hack. 

Long term, the goal is to use the core encoder code, so that encoder mapping can be used too.